app/src/components/market/sections/market_create/steps/items/ask_question_step.tsx

```diff
@@ -90,6 +85,16 @@ const ButtonWithReadyToGoStatus = styled(Button)<{ readyToGo: boolean }>`
   ${props => props.readyToGo && ButtonWithReadyToGoStatusCSS}
 `
 
+const CategoryImportWrapper = styled.div`
+  border-bottom: 1px solid ${props => props.theme.borders.borderDisabled};
+  margin-left: -${props => props.theme.cards.paddingHorizontal};
+  margin-right: -${props => props.theme.cards.paddingHorizontal};
+  padding: 0 ${props => props.theme.cards.paddingHorizontal};
+  padding-bottom: 20px;
+  & > * + * {
```

Comment:
See other comment about referencing an element in this manner. If there is a simpler way, let's do it to avoid any bugs arising if we update the markup.

Reply:
@KadenZipfel , I think it's not a good idea to add "margin" to each child components.

Reply:
After looking at the code again, this implementation is reasonable